<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="leftSide" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:lightgrey;</string>
    </property>
    <widget class="QPushButton" name="page1_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Day Todo</string>
     </property>
    </widget>
    <widget class="QPushButton" name="page2_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>日程概览</string>
     </property>
    </widget>
    <widget class="QPushButton" name="page3_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>我的课表</string>
     </property>
    </widget>
    <widget class="QPushButton" name="page4_Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>我的森林</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="underSide" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>509</y>
      <width>621</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:gray;</string>
    </property>
    <widget class="QPushButton" name="plantButton">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开始种树</string>
     </property>
    </widget>
    <widget class="QPushButton" name="whiteNoiseButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>白噪音</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="rightScrollArea">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>201</width>
      <height>511</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>199</width>
       <height>509</height>
      </rect>
     </property>
     <widget class="QPushButton" name="hideButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="descriptionEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>添加描述</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>161</width>
        <height>23</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>日程名称</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>161</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2100</year>
        <month>12</month>
        <day>29</day>
       </datetime>
      </property>
      <property name="minimumDate">
       <date>
        <year>2000</year>
        <month>9</month>
        <day>14</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>621</width>
      <height>511</height>
     </rect>
    </property>
    <widget class="QWidget" name="page1">
     <widget class="QDateEdit" name="mainDateEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2100</year>
        <month>12</month>
        <day>29</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>8</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2000</year>
        <month>9</month>
        <day>14</day>
       </datetime>
      </property>
      <property name="minimumDate">
       <date>
        <year>2000</year>
        <month>9</month>
        <day>14</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="mainNameEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>561</width>
        <height>41</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>在此添加内容，按回车创建事件</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>561</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QScrollArea" name="matterScrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>557</width>
            <height>317</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page2"/>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>leftSide</zorder>
   <zorder>underSide</zorder>
   <zorder>rightScrollArea</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
